src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +16,40 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    loadLastSession(swagger);
     showLoginUI(swagger);
 }
 
+const loadLastSession = (swagger) => {
+    if (isAuthorized(swagger))
+        return;
+
+    const lastToken = window.localStorage.getItem(TOKEN_SWAGGER_NAME);
+    if (!lastToken)
+        return;
+
+    const obj = generateAuthorizeObjectForSwagger(lastToken);
+    swagger.authActions.authorize(obj);
+}
+
 const overrideSwaggerAuthorizeEvent = (swagger) => {
     const auth = swagger.authActions.authorize;
-    swagger.authActions.authorize = (args) => attachReloadPageToEvent(swagger, auth, args);
+    swagger.authActions.authorize = async (args) => {
+        const result = await auth(args);
+        window.localStorage.setItem(TOKEN_SWAGGER_NAME, result.payload.bearerAuth.value);
```

Comment:
shouldn't we check `result.payload.bearerAuth.value` for accessing undefined properties?

Reply:
it's not necessary because our code creates the object. the only way may object change when the method of authentication changes to another thing like OAuth